src/main/java/create_task_use_case/CreateTaskInteractor.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Create Task Interactor Class.
+ * Author: Fardin Faruk
+ * Modified By: Fardin Faruk
+ * Created: Nov 19, 2022
+ * Last Modified: Nov 26, 2022
+ */
+package create_task_use_case;
+
+import entities.Task;
+import entities.TaskFactory;
+
+public class CreateTaskInteractor implements CreateTaskInputBoundary {
+    final CreateTaskDsGateway createTaskDsGateway;
+    final CreateTaskOutputBoundary createTaskOutputBoundary;
+    final TaskFactory taskFactory;
+
+    public CreateTaskInteractor(TaskFactory taskFactory, CreateTaskOutputBoundary createTaskOutputBoundary,
+                                CreateTaskDsGateway createTaskDsGateway){
+        this.taskFactory = taskFactory;
+        this.createTaskOutputBoundary = createTaskOutputBoundary;
+        this.createTaskDsGateway = createTaskDsGateway;
+    }
+
+
+    @Override
+    public CreateTaskOutputData create(CreateTaskInputData createTaskInputData) {
+        if (createTaskDsGateway.existsByTitle(createTaskInputData.getTitle(), createTaskInputData.getDayID())){
+            return createTaskOutputBoundary.prepareFailView("Task with the same name already exists.");
```

Comment:
I believe returning DayID will not be very clear to the general user, perhaps I return the name of the day in which case I'll need to create an enum or something of that sort.

Reply:
It may be useful to make an enum outside of the use case packages so we can share this approach to formatting a response message across all use cases.